include/Options/Business/Sync.php

```diff
@@ -37,16 +37,14 @@ public static function key(): string {
 	 *
 	 * @return array Partial schema.
 	 *
-	 * @phpstan-return array{type: 'array', items: array{type: SchemaType, enum: non-empty-array<non-falsy-string>}}
+	 * @phpstan-return array{type: 'array', items: array{type: SchemaType, enum: non-empty-list<non-falsy-string>}}
 	 */
 	protected function get_data_structure(): array {
-		/** @phpstan-var non-empty-array<non-falsy-string> */
-		$enum = array_keys( \PLL_Settings_Sync::list_metas_to_sync() );
 		return array(
 			'type'  => 'array',
 			'items' => array(
 				'type' => $this->get_type(),
-				'enum' => $enum,
+				'enum' => \PLL_Settings_Sync::get_synchronizable_elements(),
```

Comment:
Could you move all the related changes to a separate PR? I'd like to cherry-pick this PR to the 3.6.x branch to avoid the notices in the next minor release.

Moreover, instead of duplicating code from `list_metas_to_sync()` to `get_synchronizable_elements()`, I suggest to prevent loading the translation as we don't need them anyway.
I did not tests but I expect something like this to work:
```php
add_filter( 'lang_dir_for_domain', '__return_false' ); // Prevents loading the translations too early.
$enum = array_keys( \PLL_Settings_Sync::list_metas_to_sync() );
remove_filter( 'lang_dir_for_domain', '__return_false' );
```

Reply:
Reverted in [795fae4](https://github.com/polylang/polylang/pull/1550/commits/795fae4ca38df08eef6eb6c72059949762d7c758).